iOS/XamDialogs/component/component.yaml

```diff
@@ -0,0 +1,30 @@
+%YAML 1.2
+---
+
+name: XamDialogs
+id: xamdialogs
+summary: Custom dialogs for picking dates and from simple lists
+publisher: Xamarin Inc
+license: ../License.md
+icons: 
+  - xamdialogs_128x128.png
+  - xamdialogs_512x512.png
+publisher-url: https://go.microsoft.com/fwlink/?linkid=865313
+src-url: https://go.microsoft.com/fwlink/?linkid=865313
+libraries: 
+  ios-unified: 
+    - ../output/XamDialogs.dll
+details: Details.md
+screenshots: 
+  - screenshots/Screen1.png
+getting-started: GettingStarted.md
+version: 1.1
+samples: 

```

Comment:
Update sample to use Nuget with removeProject and instalNuget.  local-nuget-repo is needed too.